index.d.ts

```diff
@@ -77,6 +82,93 @@ declare const terminalImage: {
+	@param options.height - Optional: Custom image height. Can be set as percentage or number of rows of the terminal. It is recommended to use the percentage options.
+	@param options.maximumFramerate - Optional: Maximum framerate to render the GIF.
+	@param options.updateLog - Optional: Custom handler which is run for each frame of the gif.
+	@param options.updateLog.done - Optional: Custom handler which is run when the animation playback is stopped.
+	@returns A function that can be called to stop the gif animation.
+
+	@example
+	```
+	import terminalImage = require('terminal-image');
+	import delay = require('delay');
+	const fs = require('fs').promises;
+
+	(async () => {
+	const gifData = await fs.readFile('unicorn.gif');
+	const stopAnimation = terminalImage.gifBuffer(gifData)
+
+	await delay(5000)
+	stopAnimation()
+	})();
+	```
+	*/
+	gifBuffer: (imageBuffer: Readonly<Buffer>, options?: Readonly<{
+		width?: number;
+		height?: number;
+		maximumFramerate?: number;
+		updateLog?: typeof UpdateLog;
+	}>) => () => void;
+
+	/**
+	Display gifs in the terminal.
+
+	Optionally, you can specify the height and/or width to scale the image.
+	That can be either the percentage of the terminal window or number of rows and/or columns.
+	Please note that the image will always be scaled to fit the size of the terminal.
+	If width and height are not defined, by default the image will take the width and height of the terminal.
+	It is recommended to use the percentage option.
+	You can set width and/or height as columns and/or rows of the terminal window as well.
+	By default, aspect ratio is always maintained. If you don't want to maintain aspect ratio, set preserveAspectRatio to false.
+	Each frame of the gif is by default logged to the terminal, overwriting the previous one. To change this behaviour, set updateLog to a different function. To change the code run when the animation playback is stopped, set updateLog.done to a different function.
+
+	@param imageBuffer - Buffer with the image.
+	@param options - Image rendering options.
+	@param options.width - Optional: Custom image width. Can be set as percentage or number of columns of the terminal. It is recommended to use the percentage options.
+	@param options.height - Optional: Custom image height. Can be set as percentage or number of rows of the terminal. It is recommended to use the percentage options.
+	@param options.maximumFramerate - Optional: Maximum framerate to render the GIF.
+	@param options.updateLog - Optional: Custom handler which is run for each frame of the gif.
+	@param options.updateLog.done - Optional: Custom handler which is run when the animation playback is stopped.
+	@returns A function that can be called to stop the gif animation.
+
+	@example
+	```
+	import terminalImage = require('terminal-image');
+	import delay = require('delay');
+
+	(async () => {
+	const stopAnimation = terminalImage.gifFile('unicorn.gif')
+
+	await delay(5000)
+	stopAnimation()
```

Comment:
Indentation.

Reply:
Does it need more indention or should it be removed?